sdk/go/x/auto/stack.go

```diff
@@ -644,8 +644,8 @@ func (ur *UpResult) GetPermalink() (string, error) {
 // GetPermalink returns the permalink URL in the Pulumi Console for the update
 // or refresh operation. This will error for alternate, local backends.
 func GetPermalink(stdout string) (string, error) {
-	const permalinkSearchStr = "View Live: "
-	var startRegex = regexp.MustCompile(permalinkSearchStr)
+	const permalinkSearchStr = "View Live: |Permalink: "
```

Comment:
`View Live:` is only relevant to app.pulumi.com backend. Other backends fail to get the permalink.